test/freenet/crypt/CryptByteBufferTest.java

```diff
@@ -185,7 +190,7 @@ public void testSuccessfulRoundTripInPlaceOffset() throws GeneralSecurityExcepti
             byte[] copyBuffer = buffer.clone();
             System.arraycopy(originalPlaintext, 0, buffer, header, originalPlaintext.length);
             crypt.encrypt(buffer, footer, originalPlaintext.length);
-            assertTrue(!Arrays.equals(buffer, copyBuffer));
+            assertThat(buffer, Matchers.not(equalTo(copyBuffer)));
```

Comment:
The matchers look best when they are all statically imported, so it becomes:
```suggestion
            assertThat(buffer, not(equalTo(copyBuffer)));
```